docs/install-upgrade/install.md

```diff
@@ -101,44 +101,17 @@ This control node should be given a static IP address. Either a lease or statica
 
 ### Configure Management Network
 
-The control node is dual-homed. It has a 10GbE interface that connects to the management network. 
-The other link called `external` in the `fab.yaml` file is for the customer to access the control 
-node via their local IT network. The management network is for the command and control of the 
+The control node is dual-homed.  It has a 10GbE interface that connects to the management network 
+of the switches that comprise the fabric called `mangement` in the `fab.yam called `mangement` in the
```

Comment:
some text got copied twice here and the monospace of fab.yaml is not closed, also below there's a double "for".

Below I attempted to rephrase it - by all means feel free to throw this away if you feel I'm getting too pedantic, but years of papers review in academia will do things to you 😅 

```
The control node is dual-homed; the two networks it connects to are called respectively `management`
and `external` in the `fab.yaml` file.
The `management` network is for the command and control of the switches that comprise the fabric. It
can be a simple broadcast domain with layer 2 connectivity. The management network is not accessible
to machines or devices not associated with the fabric; it is a private, exclusive network. The control
node has a 10GbE interface that connects to the management network, and it runs a DHCP and a small http
server.
The `external` network on the other hand is for the customer to access the control node via their local
IT network. It is meant to give ssh access to the control node host operating system.
```

Reply:
It is good to hear the academic pedantry in stereo :smile: I welcome the feedback.